doc/classes/HTTPRequest.xml

```diff
@@ -14,10 +14,18 @@
 		    add_child(http_request)
 		    http_request.connect("request_completed", self, "_http_request_completed")
 
-		    # Perform the HTTP request. The URL below returns some JSON as of writing.
+		    # Perform a GET request. The URL below returns JSON as of writing.
 		    var error = http_request.request("https://httpbin.org/get")
 		    if error != OK:
-		        push_error("An error occurred in the HTTP request.")
+				push_error("An error occurred in the HTTP request.")
```

Comment:
Wrong indentation, we use 4-space indentation in code blocks (but the part which is XML indentation should still be tabs).